source/components/molecules/Card/Card.js

```diff
@@ -269,49 +288,80 @@ Card.Image = ({ firstChild, lastChild, ...props }) => (
  * Renders a progress bar
  * @param {props} props
  */
-Card.Progressbar = ({ children, firstChild, lastChild, colorSchema, ...props }) => (
+Card.Progressbar = ({
+  children,
+  firstChild,
+  lastChild,
+  colorSchema,
+  ...props
+}) => (
   <Outset lastChild={lastChild} firstChild={firstChild}>
     <CardProgressbar rounded colorSchema={colorSchema} {...props} />
   </Outset>
 );
 
+/**
+ * Renders a bullet list
+ * @param {props} props
+ */
```

Comment:
No need to add the obvious :)
ie. the comment should be removed.

Reply:
Will not change due to the whole file look like this.